### ISSUE TITLE
Hard to follow network-traffic troubleshooting instructions

### ISSUE THREAD

**Issue by sergeykad** [GoogleContainerTools/jib#4039]
**Environment**:

- *Jib version:* 3.3.2
- *Build tool:* Gradle 8.1
- *OS:* Ubuntu 22.04.2 LTS

**Description of the issue**:
The Jib troubleshooting involves running Gradle with `--no-daemon` which is challenging to do. 
See  https://github.com/gradle/gradle/issues/16110 

https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#how-can-i-examine-network-traffic

**Expected behavior**:
Troubleshooting will not involve running with `--no-daemon` .
Actually, after some additional testing, I discovered that adding `--no-daemon`  is not required, but one must use `--info` flag. Maybe the documentation needs to be updated?

**Comment by chanseokoh** [comment]
As you noted, `--no-daemon` is actually unnecessary now. It was there to make sure that you don't use a running Gradle daemon in a broken state. (A very old version of Jib could apply a wrong logging settings on a daemon.)

I am sure `--info` wasn't needed before, but if it does so now, yeah, I agree the doc should mention it. For example, "you may also need to also `--info` or `--debug`." At least the current doc suggests to set `--debug --stacktrace`.

**Comment by chanseokoh** [comment]
> The Jib troubleshooting involves running Gradle with `--no-daemon` which is challenging to do.
> See [gradle/gradle#16110](https://github.com/gradle/gradle/issues/16110)

Actually, gradle/gradle#16110 is not an issue. It doesn't matter if `--no-daemon` forks a JVM or not for our purposes. The point of `--no-daemon` is to not use the currently running daemon.

And yeah, I verified that `--info` is needed at least. I'll create a PR to update the doc.

**Comment by sergeykad** [comment]
@chanseokoh You may want to consider removing `--no-daemon` as well. It does not really change anything. See https://github.com/gradle/gradle/issues/25272#issuecomment-1584484093

**Comment by chanseokoh** [comment]
> It does not really change anything.

You have a misunderstanding. See my comments here:

https://github.com/gradle/gradle/issues/2660#issuecomment-601854215
https://github.com/gradle/gradle/issues/2660#issuecomment-602628786

Here, what matters is not about whether you use a daemon but whether you reuse the existing daemon. In you case (in most cases in my experience), it does fork a single-use (short-lived) daemon and doesn't use the existing daemon, which is the intention for our logging purposes.

So we do want `--no-daemon`. There are other workarounds like running `gradle --stop` first, but I think documenting `--no-daemon` is the most efficient and error-prone approach.

**Comment by sergeykad** [comment]
I see. Thanks for clarifying. 
I was under the impression that changing the daemon JVM configuration will force the creation of a new one unless there is already an existing daemon with this configuration.

In any case, I thought that since `--no-daemon` should be added to see the logs, it is important that the build is running without a daemon, and just adding `--no-daemon` is not enough for it in the recent Gradle versions.
